nlpretext/token/tokenizer.py

```diff
@@ -48,6 +53,19 @@ def get_lang_model(self):
         return self.model.lang
 
 
+def _load_spacy_model(model: str) -> Any:
+    try:
+        return spacy.load(model)
+    except OSError:
+        raise LanguageNotInstalledError(
+            (
+                f'Model {model} is not installed. '
+                'Find the latest release at '
```

Comment:
The error message might be a little bit confusing and doesn't tell how to solve the issue. 
I would adapt a little bit the message:
```
msg = f'Model {model} is not installed. To install, run:'
msg += f"python -m spacy download {model}"
```

Reply:
Much better! Done!